client/state/selectors/get-sites.js

```diff
@@ -20,7 +20,7 @@ export default createSelector(
 	( state ) => {
 		const primarySiteId = getPrimarySiteId( state );
 		const [ primarySite, sites ] = partition( state.sites.items, { ID: primarySiteId } );
-		return map( concat( primarySite, sites ), site => getSite( state, site.ID ) );
+		return map( concat( primarySite, sortBy( sites, [ 'name', 'URL' ] ) ), site => getSite( state, site.ID ) );
```

Comment:
Yeah, I was using the _auto-curried iteratee-first data-last_ version of `sortBy` from `lodash/fp`, sorry.

Now that you mention it, `primarySite` is actually `[{}]` or `[]`, so, it'll need the spread. Maybe not the best name?